dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.39/V2_39_14__Add_referral_column_to_relationshiptype.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE relationshiptype ADD COLUMN IF NOT EXISTS referral BOOLEAN;
+
+UPDATE relationshiptype SET referral = FALSE WHERE referral IS NULL;
+
+ALTER TABLE relationshiptype ALTER COLUMN referral SET NOT NULL;
```

Comment:
would it make sense to add a default?

Reply:
I had the same doubt when I was creating the script but then I saw that in all similar case we are doing in this way.
Maybe @stian-sandvold would know more and would know if there is any reason not to have default values on the DB.
Otherwise I am happy to change it

Reply:
As far as I know, there is no practical reason for not adding a default, other than the fact that we rarely do it. You can add it as a default instead if you want :)

Reply:
Changed to default